### PR TITLE
WDIP-21: Fix handling of Civil Disorder in beta UI

### PR DESCRIPTION
**Pull request by adamlerer**
Handle Civil Disorder in beta UI, based on Markus suggestion. The implementation could be a bit more elegant, but it's alright...

https://user-images.githubusercontent.com/5702157/175790559-dccff2cf-b535-4db9-ac11-abff76dcc7c2.mov

### CODE REVIEW

**Review by utiq — APPROVED** [review]

File: api.php [hunk 1]
```diff
@@ -467,6 +466,29 @@ public function run($userID, $permissionIsExplicit) {
...
+/**
+ * API entry game/setbackfromleft
+ */
+class SetBackFromLeft extends ApiEntry {
+	public function __construct() {
+		parent::__construct('game/setbackfromleft', 'JSON', '', array('gameID','countryID'));
+	}
+	public function run($userID, $permissionIsExplicit) {
+		global $Game, $DB;
+		error_log("000");
```
Comment (lightvector) [inline api.php:478]: Should this and the other error_logs below be here?

File: objects/member.php [hunk 2]
```diff
@@ -243,5 +243,62 @@ function send($keep, $private, $text, $fromCountryID=null)
 			$this->userID, $this->gameID, 'Game',
 			$keep, $private, $text, $this->Game->name, $this->gameID);
 	}
+
```
Comment (lightvector) [inline objects/member.php:246]: Is the movement of this code between files a no-op with regards to any other call sites? Does the old UI still work as it did before regarding civil disorder and rejoining?

Comment (adamlerer) [reply objects/member.php:246]: I tried it and it seems to work as before, but I can't say for sure.

File: api.php (hunk 1 above)
Comment (adamlerer) [reply api.php:478]: Thanks for catching this.

File: objects/member.php (hunk 2 above)
Comment (adamlerer) [reply objects/member.php:246]: I believe it's a no-op because `boardMember` here is a subclass of `Member`

**Pull request merged**